coreutils/Makefile

```diff
@@ -300,10 +300,10 @@ ${COREUTILS_NOLINK_DIR}:
 	# Generate line coverage information
 	if [ "${ENABLE_COVERAGE}" != "OFF" ] ; then \
 		${CURDIR}/../utils/coreutilscov.sh "$*" "$$OUTPUT_DIR" ${COREUTILS_COV_MAX_TIME}
-	fi
-	GCDA_FILES=`find "${COREUTILS_COV_DIR}" -name "*.gcda"` ; \
-	if [ ! -z "$$GCDA_FILES" ] ; then \
-		rm -f $$GCDA_FILES ; \
+		GCDA_FILES=`find "${COREUTILS_COV_DIR}" -name "*.gcda"` ; \
+		if [ ! -z "$$GCDA_FILES" ] ; then \
+			rm -f $$GCDA_FILES ; \
+		fi
```

Comment:
@rasoolmaghareh This line also has to end with ; \
to continue to the next line.